### ISSUE TITLE
[Question] Can I edit default columns shown in omniboard?

### ISSUE THREAD

**Issue by mf-giwoong-lee** [vivekratnavel/omniboard#274]
Omniboard supports adding custom columns. But this job is manual. 

I want to customize the default columns to check quickly the important hyper-parameters like learning rate, batch size etc.

It can be possible in python code level or omniboard configuration file?

**Comment by Haetsal-Lee** [comment]
As far as I know, currently, your custom settings like column orders are stored in Local-storage of a browser.

If you are using the Chrome, you can use the[LocalStorage-Manager ](https://chrome.google.com/webstore/detail/localstorage-manager/fkhoimdhngkiicbjobkinobjkoefhkap) plugin, to save and restore your settings.
